src/components/Select/styles.styl

```diff
@@ -23,14 +26,13 @@ select-date-height-small=2.75rem
     cursor pointer
     color coolGrey
 
+.Select .Select__Icon
+    position absolute
+    top 50%
+    transform translateY(-50%)
+    right 0.875rem
```

Comment:
It seems weird to me to have the arrow in absolute position. Maybe we could use flexbox to center the icon vertically ?

Reply:
We need it to be on top of the select, so clicking on it actually clicks on the select. If we use flexbox, they will be next to each other, no ? Do you have something in mind ?

Reply:
Ah yes OK I see.